tensorflow/python/keras/layers/preprocessing/image_preprocessing.py

```diff
@@ -466,6 +467,8 @@ class RandomTranslation(PreprocessingLayer):
         The input is extended by wrapping around to the opposite edge.
       - *nearest*: `(a a a a | a b c d | d d d d)`
         The input is extended by the nearest pixel.
+    fill_value: a float represents the value to be filled outside the
```

Comment:
Put `fill_value` at the end

Reply:
do you mean also change the input signature for `__init__`?

```python
  def __init__(self,
               height_factor,
               width_factor,
               fill_mode='reflect',
               interpolation='bilinear',
               seed=None,
               name=None,
               fill_value=0.0,
               **kwargs):
```